splinepy/bezier.py

```diff
@@ -4,6 +4,61 @@
 class BezierBase(spline.Spline):
     """Bezier Base. Contain extra operations that's only
     available for bezier families.
+
+    Beziers are a special type of spline where the basis functions have
+    a global support. The basis functions are given by the so-called
+    *Berstein polynomials*
+
+    .. math::
+            B_{i;p}(u) = \\binom{p}{i} u^i (1-u)^{p-i}
+
+    The parametric domain of a Bezier spline is always a multi-dimensional
+    hypercube, that is :math:`\\Omega_{param}=[0,1]^{N_{param}}`.
+    We can distinguish between different types of splines depending on the
+    dimension of the parametric space.
+
+    1. A spline of degree :math:`p` with control points
+    :math:`P_i\\in\\mathbb{R}^{N_{phys}}` and a one-dimensional parameter space
+    (i.e., :math:`N_{param}=1`) corresponds to a line embedded into the
+    physical space:
+
+    .. math::
+            S^B(u) = \\sum_{i=0}^{l} B_{i;p}(u) P_i
```

Comment:
This is just a minor change, but I suggest to stick to the general naming convention used by GE and also NURBS book, where curves are `C`, Surfaces `S` and Volumes `V`, higher dimensional objects are then written as `M` for multivariate.

I don't mind if you stick to the current notation, just wanted to put it out there.

Reply:
Alright, I can change that.